<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>QtClippingDialogUi</class>
 <widget class="QWidget" name="QtClippingDialogUi" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>457</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>1</hsizetype>
    <vsizetype>1</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Clip Planes</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>1</hsizetype>
       <vsizetype>1</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Sweep</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="2" >
       <widget class="QLineEdit" name="sweepRangeLineEdit" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="text" >
         <string>1</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>1</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>1</hsizetype>
          <vsizetype>5</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Sweep range</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3" >
       <widget class="QSlider" name="sweepRangeSlider" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>1</hsizetype>
          <vsizetype>5</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum" >
         <number>-100</number>
        </property>
        <property name="maximum" >
         <number>100</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>1</hsizetype>
       <vsizetype>1</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Plane(s)</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>1</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>10</width>
          <height>1</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>1</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1" >
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="1" column="0" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>1</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1" >
         <widget class="QPushButton" name="setButton" >
          <property name="text" >
           <string>Set</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2" >
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="1" column="2" >
           <widget class="QLineEdit" name="normalXLineEdit" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>1</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="text" >
             <string/>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_2" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>1</hsizetype>
              <vsizetype>5</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Normal</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4" >
           <widget class="QLineEdit" name="normalZLineEdit" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>1</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text" >
             <string/>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="label_3" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>1</hsizetype>
              <vsizetype>5</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Slice width</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2" >
           <widget class="QLineEdit" name="sliceWidthLineEdit" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>1</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="text" >
             <string/>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight</set>
            </property>
           </widget>
          </item>
          <item row="0" column="3" >
           <widget class="QLineEdit" name="positionYLineEdit" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>1</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="text" >
             <string/>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight</set>
            </property>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="label" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>1</hsizetype>
              <vsizetype>5</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Position</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" >
           <widget class="QLineEdit" name="positionXLineEdit" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>1</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="text" >
             <string/>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight</set>
            </property>
           </widget>
          </item>
          <item row="0" column="4" >
           <widget class="QLineEdit" name="positionZLineEdit" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>1</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text" >
             <string/>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="3" >
           <widget class="QLineEdit" name="normalYLineEdit" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>1</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="text" >
             <string/>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="2" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>10</width>
          <height>1</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>10</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>1</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
